Content: Unfettered campaign Act 1 (#6416)
endless-sky/endless-sky: data/hai/hai.txt

	"Sea Scorpion"
	"Shield Beetle (Unfettered Shipyards)"
	# Has "Solifuge" and "Violin Spider" added by event.
	# Technically has the specs for the "Ladybug" but not the resources or interest to build it.

> 

Reviewer: ```suggestion

```

	"Asteroid Scanner"
	"Outfit Scanner"

> 

Reviewer: ```suggestion

```

# When possible add all outfits not here but in the Hai shipyards at 1.5x the cost.
# Includes the ion cannon, the reverse thrusters, the Valley Batteries and the biggest engines.
outfitter "Unfettered"
	"Pulse Turret"
	"Chameleon Anti-Missile"
	"Hai Tracker Pod"
	"Tracker Storage Pod"
	"Bunk Room"
	"Outfits Expansion"
	"Hai Diamond Regenerator"
	"Hai Gorge Batteries"
	"Hai Ravine Batteries"
	"Boulder Reactor"
	"Pebble Core"
	`"Benga" Atomic Thruster`
	`"Benga" Atomic Steering`
	`"Biroo" Atomic Thruster`
	`"Biroo" Atomic Steering`
	`"Bondir" Atomic Thruster`
	`"Bondir" Atomic Steering`
	`"Bufaer" Atomic Steering`

> 

Reviewer: ```suggestion

```

outfitter "Mera's Lab"
	"Hai Tracker"
	"Chameleon Anti-Missile"
	"Outfits Expansion"
	"Cargo Expansion"
	"Hai Octopus Jammer"
	"Value Detector"
	"Outfit Scanner"
	"Cargo Scanner"
	"Hyperdrive"
	"Hai Williwaw Cooling"
	"Pebble Core"
	"Supercapacitor"
	"Pulse Rifle"
	"Asteroid Scanner"
	"Fuel Pod"

> 

Reviewer: ```suggestion

```

	"Cargo Scanner"
	"Sand Cell"

> shipyard "Unfettered"

Reviewer 2: It's a breaking change to rename these shipyards and outfitters.

Author: oh sorry, was not aware of that, because the map is part of the data folder and reference to said shipyard is only in the data folder, not in the save files, I'll revert that change then I guess

but still it feels pretty weird to have them named Basics and give them Shield Beetles x)

Reviewer 2: It's a game object that can be extended by the event system. It is also available for use by any plugin (just like any plugin can use an existing ship or outfit in it's content). Renaming them means any plugin would be referring to a now-missing list of ships or outfits.

W.r.t. adding things to it, if an item doesn't make thematic sense, then introduce a new outfitter where the item will make thematic sense.

	word
		"!"
		"."

phrase "hostile unfettered challenger"
	word
		"Stop "
		"Quit "
	word
		"shaming yourself"
		"running away"
		"thinking too much"
		"recharging your shields"
		"hiding in fear"
	word
		" and come "
		" and come try and "
	word
		"defeat "
		"fight "
	word
		"me."
	word
		" Now is the time to reveal ourselves in the glory of battle"
		" Show me what you got"
		" That's more like it"
		" Time for a fair fight"
	word

> 		"."

Reviewer 3: ```suggestion
```
I don't think this fits here.

		"!"

phrase "hostile unfettered challenger"
	word
		"Now is not the time for talk"
		"Yes! Fight me"
		"Our battle will be sang in legends for years to come"

> 		"Do not try to distract me with hails."

Reviewer 3: ```suggestion
		"Do not try to distract me with hails"
```

		"You succeeded in "
		"You managed to have "
		"Somehow, you have "
		"I must admit you have "
	word
		"beaten "
		"defeated "
		"ashamed "
		"disabled "
		"revealed my weakness% "
	word
		"me in a% "
	word
		"glorious "
		"pathethic "
		"exceptional "
		"grandiose "
		"quick "
		"strategic "
	word
		"battle"
		"combat"
		"duel"
		"fight"
		"challenge"
	word
		"."
		"!"
	word
		" What more do you want from me?"
		" Do not presume you may board me."
		" Do not even think about capturing my ship."
		" Now it is over, time to lay down your weapons."
		" I accept my fate, take what is rightfully yours."
		" Great stories will be told of our battle!"
		" You have the right to plunder."
		" It would seem you are as strong as some pretended, perhaps even more."
		" Finish me! I would rather die than have to live with this insult."
	replace
		"% me" ""
		"a% e" "an e"
		"a%" "a"

phrase "unfettered duelling"
	word
		"Do not presume to interrupt us"
		"I am in the middle of a duel, there is no time for petty talk"
		"This is an official challenge, do not intervene"
		"Under no circumstances are you to interrupt us, or you will suffer the wratch of the Unfettered"
		"Stop distracting me"

> 		"If you try anything the whole fleet will be on you"

Reviewer 3: ```suggestion
		"If you try anything, the whole fleet will be on you"
```

	word
		"! "
	word
		"Go away now, I need to focus on the duel"
	word
		"!"
		"."

phrase "friendly mercenary"
	word
		"What is your command"
		"Orders"
		"Anything I can do for you"
		"Who should I shoot"
		"Covering fire, where"
		"Yes"
		"Change of plan"

> 		"I am ready"

Reviewer 3: ```suggestion
```
`I am ready, Boss?` shouldn't be a question, unless the mercenary is really insecure.

	word
		", "
	word
		"Boss"
		"Leader"
		"Alpha"
		"First Mercenary"
		"Commander"
		"Captain"
	word
		"?"

phrase "disabled mercenary"
	word
		"I await judgment."
		"Do not expect me to beg."
		"It seems I was proven... weak."

> 		"I can pay you... If you would only lean me a hand."

Reviewer 3: ```suggestion
		"I can pay you... If you would only lend me a hand."
```

		"Truly a shameful display on my part."
		"I have executed the former captain for his incompetence."
		"Leave me alone, I do not deserve any help."

phrase "enemy mercenary"
	word
		"You have "
		"You possess "
		"On your ship is "
	word
		"something "
		"an item "
		"a possession "
	word
		"precious to us"
		"valuable to us"

> 		"of mine "

Reviewer 3: ```suggestion
		"of mine"
```
Should it be `mine` or `ours`?

> 		"that we want "

Reviewer 3: ```suggestion
		"that we want"
```

	word
		"."
		"!"
	word
		" Surrender it"
		" Drop it in the next spaceport"
		" Give it to me and your death will be quick."
		" Nemo Me Impune Lacessit"
		" Sell it on a lawless world"

> 		" Give it back, scum"

Reviewer 3: `Give it back` only works with `of mine`; maybe it should be `Give it to us` or `Hand it over`?

	word
		"."
		"!"

phrase "enemy mercenary"
	word
		"Nemo Me Impune Laccessit!"
		"You should run if you still hope to see the sun rise."
		"We will never stop hunting you!"

> 		"You do not understand of value of what you have stolen."

Reviewer 3: ```suggestion
		"You do not understand the value of what you have stolen."
```

		"Mere children cannot be allowed to hold on to such power!"

phrase "cerberus hails"
	word
		"We do not accept your surrender!"
		"You stand no chance!"
		"Even if you take me down, my master will make quick work of you."
		"Do not attempt to subdue me."
		"I obey only the masters."

> 		"They hold my soul! Soon, they will hold your as well."

Reviewer 3: ```suggestion
		"They hold my soul! Soon, they will hold yours as well."
```

		"Die!"
		"You have been allowed to live long enough. Now I come to cut the thread of your life."
		"I may be a pawn, but at least I am on a greater path."

> 		"You may never hope to achieve half of what I did."

Reviewer 3: ```suggestion
		"You may never hope to achieve half of what I have done."
```
